MoltenVKShaderConverter/MoltenVKSPIRVToMSLConverter/SPIRVToMSLConverter.cpp

```diff
@@ -237,7 +237,7 @@ MVK_PUBLIC_SYMBOL bool SPIRVToMSLConverter::convert(SPIRVToMSLConverterContext&
 		mslOpts.aux_buffer_index = context.options.auxBufferIndex;
 		mslOpts.enable_point_size_builtin = context.options.isRenderingPoints;
 		mslOpts.disable_rasterization = context.options.isRasterizationDisabled;
-		mslOpts.swizzle_texture_samples = true;
+		mslOpts.swizzle_texture_samples = false;
```

Comment:
As per the [discussion](https://github.com/KhronosGroup/MoltenVK/issues/306#issuecomment-442178227) in #306...we'll probably make `mslOpts.swizzle_texture_samples = false;` the default in the end...but I would like to do so in a separate PR that contains flexibility for easily changing it. So can you remove this from this PR, please?

Reply:
Yes, I will revert back to true again. Keep in mind this has a huge impact on performance in our case, e.g. we get 30 fps instead of 25 fps.